.changes/v2.10.0/543-notes.md

```diff
@@ -0,0 +1 @@
+* Skip faulty check in test `TestAccVcdVAppVmDhcpWait`
```

Comment:
Why the check is faulty and why are we not fixing the cause? And from other perspective, if it's faulty for long-term, why we are not removing the test instead of skipping?

CC: @Didainius

Reply:
It's faulty because VCD is returning a random IP instead of the intended one. Our software does the correct thing.  I left the test in place because we want to reactivate when VCD behavior is fixed.

Reply:
added a conditional skip depending on an environment variable